pages/content/amp-dev/documentation/components/reference/amp-3d-gltf@ar.md

```diff
@@ -0,0 +1,121 @@
+---
+$category@: media
+formats:
+  - websites
+teaser:
+  text: Displays GL Transmission Format (gITF) 3D models.
+---
+
+<!--
+حقوق الطبع والنشر 2018 لصالح "مؤلفو شفرة HTML لصفحات AMP". جميع الحقوق محفوظة.
+
+تم الترخيص بموجب ترخيص Apache، الإصدار 2.0 (المشار إليه لاحقًا باسم "الترخيص")؛ ولا يحق لك استخدام هذا الملف إلا بما يتوافق مع الترخيص.
+ويمكنك الحصول على نسخة من الترخيص على الصفحة
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+ما لم يكن مطلوبًا بموجب القانون الساري أو تمت الموافقة عليه كتابةً، يتم توزيع البرنامج الذي يتم توزيعه بموجب الترخيص "كما هو"، وبدون أية ضمانات أو شروط من أي نوع، سواء كانت صريحة أو ضمنية.
+راجِع الترخيص للاطّلاع على اللغة المحددة التي تحكم الأذونات والقيود بموجب الترخيص.
+-->
+
+#amp-3d-gltf
```

Comment:
do these headlines actually print correctly without a space after the `#`? In any case, would prefer a space if possible, formatting wise.